code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -45,6 +44,7 @@
 	unique_name = 1
 	gold_core_spawnable = HOSTILE_SPAWN
 	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
+	see_in_dark = 8
```

Comment:
this isnt documented or put into the changelog, why change this from 4 to 8?

Reply:
Ah, I forgot about this.  Honestly, this change is more of a convenience than anything else, limited vision in the dark isn't something considered in balancing the spiders, and this vision AOE applies extremely inconsistently.